pom.xml

```diff
@@ -0,0 +1,215 @@
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.0</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/plugins/*</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-pmd-plugin</artifactId>
+                <version>3.12.0</version>
+                <configuration>
+                    <rulesets>
+                        <ruleset>src/main/resources/plugins/pmd-ruleset.xml</ruleset>
+                    </rulesets>
+                </configuration>
+            </plugin>
+        </plugins>
+    </reporting>
+    <packaging>jar</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>com.avispl.symphony.api</groupId>
+            <artifactId>symphony-api-dal</artifactId>
+            <version>${symphonyApiVersion}</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-dal-4.9.0.00409.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.avispl.symphony.dal</groupId>
+            <artifactId>symphony-dal-commons</artifactId>
+            <version>${symphonyApiVersion}</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-dal-commons-4.9.0.00409.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.avispl.symphony.api</groupId>
+            <artifactId>symphony-api-commons</artifactId>
+            <version>${symphonyApiVersion}</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-commons-4.9.0.00409.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>org.icmp4j</groupId>
+            <artifactId>icmp4j</artifactId>
+            <version>1018</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/icmp4j-1018.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.sshd</groupId>
+            <artifactId>sshd-core</artifactId>
+            <version>2.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+            <version>5.6.0-M1</version>
+            <scope>test</scope>
+        </dependency>
+
+
+        <!--dependencies of symphony-dal-commons-->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>4.2.4.RELEASE</version>
+        </dependency>
```

Comment:
All dependencies provided by Symphony should be explicitly marked with <scope>provided</scope>

Reply:
Got it, thanks